app/views/linkout.scala.html

```diff
@@ -0,0 +1,34 @@
+@import db.ModelService
+@import db.impl.access.UserBase
+@import ore.{OreConfig, OreEnv}
+@(remoteUrl: String)(implicit messages: Messages, session: Session, requestHeader: RequestHeader, request: Request[_] = null,
+        service: ModelService, config: OreConfig, users: UserBase, env: OreEnv)
+
+@bootstrap.layout(messages("general.linkout.title")) {
+
+    <div class="container" style="margin-top: 90px;">
+        <div class="row">
+            <div class="col-md-6 col-md-offset-3">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+	                    <h3 class="panel-title">
+	                        @messages("general.linkout.title")
+	                    </h3>
+                    </div>
+                    <div class="panel-body">
+                        <p>@messages("general.linkout.warning", remoteUrl)</p>
+                        <a href="/" class="pull-left unsafe-dl-back">
```

Comment:
This needs to go back to the previous page instead of going home.

Reply:
I've said previously, but I think the best approach is to
> have the page load on a new tab, and have the "get me out of here" link close tab.

Reply:
i prefer phases approach on this one, with some of the isolation stuff coming to browsers I am not sure how reliable that will be long term